connector/src/main/scala/com/microsoft/kusto/spark/datasink/RowCSVWriterUtils.scala

```diff
@@ -35,25 +36,30 @@ object RowCSVWriterUtils {
       writer.writeStringField(json)
     }
   }
+  private def tss (l:Long, timeZone: ZoneId) ={
+    LocalDateTime.ofInstant(Instant.EPOCH.plus(l, ChronoUnit.MICROS), timeZone).toString
```

Comment:
I think this method would make more sense returning a LocalDateTime. The caller can call "toString" on it.
Plus, that would be consistent with the other "writer.writeStringField" calls below.

Reply:
thats right